### ISSUE TITLE
Renaming project doesn't update project name in webhook

### ISSUE THREAD

**Issue by bjarnef** [umbraco/Umbraco.Cloud.Issues#529]
### Issue description

We have renamed one of our Umbraco Cloud projects and the Cloud URL has been updated.

However we noticed on deployment, where we get a notification in a Slack channel reading from webhook notification body, but this still contain the original project name.
https://our.umbraco.com/documentation/umbraco-cloud/Deployment/Deployment-Webhook/

`SiteUrl` property is correct, but `ProjectUrl` and `ProjectName` properties still contain to original values. Maybe there should also be a `ProjectAlias` which contained the alias and `ProjectName` instead contains the actual name (currently this contains the alias).

**Comment by bjarnef** [comment]
Hopefully some day we can also rename a project without affecting the alias: https://github.com/umbraco/Umbraco.Cloud.Issues/issues/48

Is should still we possible to rename/change alias, but could be a more hidden feature under (advanced) settings.

**Comment by sajumb** [comment]
@bjarnef 
Thanks for the report. We have a task in the backlog where we want to look into renaming the project without affecting the project url. We will look into the consequence of changing this and locate the supporting features that likewise will have to be modified to preserve compliance We did not have a task for correct handling of an existing webhook when changing the project name/URL. We have added this to the backlog.
